src/flowchem/client/component_client.py

```diff
@@ -27,10 +27,13 @@ def post(self, url, data=None, json=None, **kwargs):
     def put(self, url, data=None, **kwargs):
         """Send a PUT request. Returns :class:`Response` object."""
 
-        # Inspect the keywargs type to avoid ploblems with not str parameters
-        if kwargs["params"]:
-            for key, arg in kwargs["params"].items():
-                if type(arg) is not str:
-                    kwargs["params"][key] = str(arg)
+        # Inspect the keyargs type to avoid ploblems with not str parameters
+        try:
```

Comment:
just use kwargs.get("params")
Will return sth if this key exist and none if not -> avoids verbose try except